package.json

```diff
@@ -5,18 +5,23 @@
   "directories": {
     "lib": "lib"
   },
+  "bin": "./bin/sonarjs",
   "scripts": {
     "test": "jest",
     "clear": "rm -rf lib/*",
-    "build": "yarn clear && yarn && tsc -p ."
+    "build-mvn": "cd sonarmini && mvn clean install && cd ..",
+    "copy-jars":
```

Comment:
@inverno why new lines?

Reply:
@vilchik-elena the ways of prettier are mysterious